webapp/src/components/singlestudy/HomeView/InformationView/index.tsx

```diff
@@ -20,6 +23,15 @@ function InformationView(props: Props) {
   const [t] = useTranslation();
   const [openVariantModal, setOpenVariantModal] = useState<boolean>(false);
   const [openLauncherModal, setOpenLauncherModal] = useState<boolean>(false);
+  const enqueueErrorSnackbar = useEnqueueErrorSnackbar();
+
+  const importStudy = async (study: StudyMetadata) => {
+    try {
+      await copyStudy(study.id, `${study.name} (${t("main:copy")})`, false);
```

Comment:
Use `usePromise` from "react-use"

Reply:
don't need, there is no state update or whatsoever. It's just a call.

Reply:
alright